drizzlepac/alignimages.py

```diff
@@ -1174,42 +1141,37 @@ def update_image_wcs_info(tweakwcs_output):
                 sci_ext_dict["{}".format(sciExtCtr)] = fileutil.findExtname(hdulist, 'sci', extver=sciExtCtr)
 
         # update header with new WCS info
-        updatehdr.update_wcs(hdulist,
-                             sci_ext_dict["{}".format(item.meta['chip'])],
-                             item.wcs,
-                             wcsname=wcs_name,
-                             reusename=True,
-                             verbose=True)
+        updatehdr.update_wcs(hdulist, sci_ext_dict["{}".format(item.meta['chip'])], item.wcs, wcs_name=wcs_name,
+                                 reusename=True, verbose=True)
         if chipctr == num_sci_ext:
             # Close updated flc.fits or flt.fits file
+            # log.info("CLOSE {}\n".format(image_name))  # TODO: Remove before deployment
             hdulist.flush()
             hdulist.close()
 
             # Create headerlet
-            out_headerlet = headerlet.create_headerlet(img_name, hdrname=wcs_name, wcsname=wcs_name)
+            out_headerlet = headerlet.create_headerlet(image_name, hdrname=wcs_name, wcs_name=wcs_name)
 
```

Comment:
Please check the parameter is `wcs_name` and not `wcsname`.  I looked at _what I believe_ to be the version of stwcs we are using and the parameter is `wcsname`.